src/utils/createParams.ts

```diff
@@ -0,0 +1,38 @@
+import { RequestConfig } from '../types/RequestConfig';
+
+// convert a request config object into a query param string
+export const createParams = (config?: RequestConfig): string => {
+  if (!config) {
+    return '';
+  }
+
+  const entries = Object.entries(config);
+  if (!entries.length) {
+    return '';
+  }
+
+  let params = '?';
+
+  entries.forEach(([key, val], i) => {
+    params += key;
+    params += '=';
+
+    if (Array.isArray(val)) {
+      val.forEach((item, index) => {
```

Comment:
Could use array val.join() here for cleaner code 
** https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join **

Reply:
Good callout, done